BeatTogether.DedicatedServer.Node/InstanceFactory.cs

```diff
@@ -40,7 +40,7 @@ public sealed class InstanceFactory : IInstanceFactory
             instanceConfig.Port = (int)port!;
             instanceConfig.Secret = secret;
             instanceConfig.ManagerId = managerId;
-            instanceConfig.MaxPlayerCount = config.MaxPlayerCount;
+            instanceConfig.MaxPlayerCount = Math.Min(config.MaxPlayerCount,127); //max size of 127
```

Comment:
Stops players from being able to crash the dedicated server if they manage to get over 127 players in a lobby